fec.config.js

```diff
@@ -0,0 +1,28 @@
+const { resolve } = require('path');
+
+module.exports = {
+  appUrl: '/insights/tasks',
+  debug: true,
+  useProxy: process.env.PROXY === 'true',
+  proxyVerbose: true,
+  plugins: [],
+  ...(process.env.HOT
+    ? { hotReload: process.env.HOT === 'true' }
+    : { hotReload: true }),
+  ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
```

Comment:
Not sure it is required to keep. What is the purpose of it?

Reply:
This is hand when we want to run the app on different ports while running apps together.